cfme/tests/intelligence/reports/test_canned_corresponds.py

```diff
@@ -120,8 +120,7 @@ def test_cluster_relationships(appliance, soft_assert):
 
 
 @pytest.mark.rhv2
-@pytest.mark.meta(blockers=[BZ(1504010, forced_streams=['5.7', '5.8', 'upstream']),
-                            BZ(1571254, forced_streams=['5.8', '5.9'])])
+@pytest.mark.meta(blockers=[BZ(1571254, forced_streams=['5.8', '5.9', '5.10'])])
```

Comment:
Required : We can remove 5.8 .
@pytest.mark.meta(blockers=[BZ(1571254, forced_streams=['5.9', '5.10'])])

Reply:
@sshveta Actually, I am not a fan of that. I remember the last time we thought there wouldn't be any further 5.8 builds...

Reply:
@sshveta I think it's worth keeping 5.8 for now in case we need it.

Reply:
@jan-zmeskal : There is a separate branch 5.8-breakpoint-patched for 5.8 only.

Reply:
@sshveta True that